### PR TITLE
add getVerifiedLocationToken function

### PR DESCRIPTION
**Pull request by ShiCheng-Lu**
add `Radar.getVerifiedLocationToken()` function.

tested with example app, with google play integrity API and AppAttest API on iOS.

turn on `Fraud > Enable Android Play Integrity API + Enable iOS App Attest`, then run `getVerifiedLocationToken()` from the app.

I routed the apis to a local server to check the requests (by using SharedPreference / standardUserDefaults to set the api hosts). Verified the tokens are generated + passed to server.

### CODE REVIEW

**Comment by ShiCheng-Lu** [comment]
need to bump flutter version

File: android/src/main/java/io/radar/flutter/RadarFlutterPlugin.java [hunk 1]
```diff
@@ -1407,6 +1435,7 @@ public void run() {
                 }
                 
             } catch (Exception e) {
+                Log.e(TAG, "NULL PTR HERE");
```
Comment (ShiCheng-Lu) [inline android/src/main/java/io/radar/flutter/RadarFlutterPlugin.java:1438]: remove

File: example/android/app/build.gradle [hunk 2]
```diff
@@ -25,7 +25,7 @@ apply plugin: 'com.android.application'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion 31
+    compileSdkVersion 34
```
[the remark this answers is not in the document]
Comment (ShiCheng-Lu) [reply example/android/app/build.gradle:28]: doesn't compile oterhwise

**Comment by lmeier** [comment]
```
* /home/runner/work/flutter-radar/flutter-radar/CHANGELOG.md doesn't mention current version (3.12.0).
```
You should add an entry to the changelog (mentioning no changes necessary)

File: example/android/app/build.gradle (hunk 2 above)
[the remark this answers is not in the document]
Comment (lmeier) [reply example/android/app/build.gradle:28]: Did you try 32 and 33? This should be as low as possible

**Review by lmeier — APPROVED** [review]

**Pull request merged**